config/config.exs

```diff
@@ -52,7 +52,7 @@ config :oli,
       ),
     favicons: System.get_env("BRANDING_FAVICONS_DIR", "/favicons")
   ],
-  payment_provider: OliWeb.PaymentProviders.StripeController
+  payment_provider: System.get_env("PAYMENT_PROVIDER", "stripe")
```

Comment:
A configuration pulled from config.exs will be compile time. If the intent is a release-time config, then this value should be read from the environment in `config/releases.exs`